chromadb/db/impl/grpc/client.py

```diff
@@ -164,30 +165,22 @@ def get_segments(
     @overrides
     def update_segment(
         self,
+        collection: UUID,
         id: UUID,
-        collection: OptionalArgument[Optional[UUID]] = Unspecified(),
         metadata: OptionalArgument[Optional[UpdateMetadata]] = Unspecified(),
     ) -> None:
-        write_collection = None
-        if collection != Unspecified():
-            write_collection = cast(Union[UUID, None], collection)
-
         write_metadata = None
         if metadata != Unspecified():
             write_metadata = cast(Union[UpdateMetadata, None], metadata)
 
         request = UpdateSegmentRequest(
             id=id.hex,
-            collection=write_collection.hex if write_collection else None,
+            collection=collection.hex,
             metadata=to_proto_update_metadata(write_metadata)
             if write_metadata
             else None,
         )
 
-        if collection is None:
-            request.ClearField("collection")
-            request.reset_collection = True
```

Comment:
This was being used for the change collection path right?

Reply:
it seems like there was also planned functionality for disassociating segments with *any* collection (also only stubbed in Go) which is what this addressed